tests/integration/tier0/custom-kernel/test_custom_kernel.py

```diff
@@ -5,6 +5,8 @@
 
 SYSTEM_RELEASE = os.environ.get("SYSTEM_RELEASE")
 
+ORACLE_LATEST_ORIGINAL_KERNEL = os.popen("rpm -q --last kernel | head -1 | cut -d ' ' -f1").read()
```

Comment:
That's clever. Can we update that to the other targets too?

Reply:
That would be nicer for sure.
The question is, if we really need to do that. 
I guess for none other than Oracle Linux 8.7 will the kernel be updated very frequently however all except CentOS8.5 could still receive an update IIUIC.
WDYT @r0x0d ?

Reply:
Well, I guess you're correct. Anything else here will receive an update, so having the static form is good.

Reply:
@danmyway , @r0x0d at firs I thought the same but then I realized that the centos7 is still getting updates (in updates), see: http://mirror.centos.org/centos/7/updates/x86_64/Packages/ , the last kernel `.80` is from 10.11.2022. So we will need to do this dynamically for centos7 and ol7 I presume as well. The only one we can skip is centos8 as it is no longer supported. It can also be handy when alma and rocky comes in place.

Reply:
Yeah, that's what I have meant with receiving the updates, I guess I could have been clearer.
Alright, I will make all original kernel packages load dynamically and keep the CentOS 8.5 static, for one there won't be any update for sure and for second, as you said @kokesak we could use that as a reference for Rocky and Alma.